### DISCUSSION TITLE
Is it possible to create video effect using swiftui view?

### DISCUSSION THREAD

**Discussion by ctopher7** [HaishinKit/HaishinKit.swift#1316] [Q&A]
I would want to use a swiftUI view as video effect / filter (like monochrome or pronama filter). Is it possible to do this

**Comment by shogo4405** [comment]
Sure. Please see Example project. https://github.com/shogo4405/HaishinKit.swift/blob/main/Examples/iOSSwiftUI/Model/ViewModel.swift

**Reply by ctopher7** [reply]
the problem with that is, I need to use UIKit (such as UIImage) to create video effect.

**Reply by shogo4405** [reply]
I'm not sure.
